auth/session.go

```diff
@@ -12,9 +12,14 @@ import (
 func (icentral *ImqsCentral) setSessionCookieIfPermitOK(sessionKey string, token *authaus.Token, w http.ResponseWriter, r *httpRequest) {
 	r.token = token
 	auditUserLogAction(icentral, r, token.UserId, token.Identity, "User Profile: "+token.Identity, authaus.AuditActionAuthentication)
-	if permList, egroup := authaus.PermitResolveToList(token.Permit.Roles, icentral.Central.GetRoleGroupDB()); egroup != nil {
+	permList, egroup := authaus.PermitResolveToList(token.Permit.Roles, icentral.Central.GetRoleGroupDB())
+	if egroup != nil && permList == nil {
+		icentral.Central.Log.Errorf("setSessionCookieIfPermitOK: Failed to resolve permit, returning (%v)", egroup)
```

Comment:
```suggestion
		icentral.Central.Log.Errorf("setSessionCookieIfPermitOK: Failed to resolve permits: %v", egroup)
```

Reply:
This function does not seem to be used anymore...removing.